test_fuzzywuzzy.py

```diff
@@ -496,7 +496,32 @@ def test_dedupe(self):
         deduped_list = ['Tom', 'Dick', 'Harry']
 
         result = process.dedupe(contains_dupes)
-        self.assertEqual(result, deduped_list)
+        self.assertEqual(sorted(result), sorted(deduped_list))
```

Comment:
Agree that this may not fit the existing method purpose but figured others
might need something similar.  What about a separate method with no
backwards compatibility to consider?

Thanks,
Michael J. Bommarito II, CEO
Bommarito Consulting, LLC
Web: http://www.bommaritollc.com
Mobile: +1 (646) 450-3387



On Jul 18, 2016 20:08, "Jose Diaz-Gonzalez" <notifications@github.com>
wrote:

> In test_fuzzywuzzy.py
> <https://github.com/seatgeek/fuzzywuzzy/pull/123#discussion_r71253523>:
>
> > @@ -496,7 +496,32 @@ def test_dedupe(self):
> >          deduped_list = ['Tom', 'Dick', 'Harry']
> >
> >          result = process.dedupe(contains_dupes)
> > -        self.assertEqual(result, deduped_list)
> > +        self.assertEqual(sorted(result), sorted(deduped_list))
>
> Not sure how I feel about the change in resultset. @acslater00
> <https://github.com/acslater00> ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/seatgeek/fuzzywuzzy/pull/123/files/b8d6469179b9418d5cab9652ad6b79ca2dd3bdd9#r71253523>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFMPfMXcH96Gk9FMnk2hMxI3agoUUssks5qXBVagaJpZM4JCf9x>
> .
>